src/presenter/main-presenter.js

```diff
@@ -62,21 +73,30 @@ export default class MainPresenter {
       throw new Error(`IllegalArgumentException! expected: ${SortForm}`);
     }
 
-    if (MessageView instanceof Message) {
-      this.#MessageView = MessageView;
+    if (messageView instanceof Message) {
+      this.#messageView = messageView;
     } else {
       throw new Error(`IllegalArgumentException! expected: ${Message}`);
     }
 
+    if (infoView instanceof Info) {
+      this.#infoView = infoView;
+    } else {
+      throw new Error(`IllegalArgumentException! expected: ${Info}`);
+    }
```

Comment:
сделай функцию тернарник. Обрати внимание, что просто поставить `: throw new Error()` нельзя. нужно значение. Выход - позвать функцию.
кстати, для сипатичности  - можно написать имя
// https://codepen.io/mx-wolf/pen/XWZOKKg?editors=0012

```js
const alwaysThrow = (ctor) =>{throw new TypeError(`хочу экземпляр ${ctor.name}`)}
const ensureInstanceOf = (obj, ctor)=> obj instanceof ctor? obj : alwaysThrow(ctor);
```
и используй ее в конструкторе
```suggestion
    this.#infoView = ensureInstanceOf(infoView, Info);
```